app/models/tagging.rb

```diff
@@ -0,0 +1,4 @@
+class Tagging < ApplicationRecord
+  belongs_to :job
+  belongs_to :tag
+end
```

Comment:
I would have loved to see you use a test to drive the development of a join table.